tedana/decomposition/eigendecomp.py

```diff
@@ -143,21 +143,22 @@ def tedpca(catd, OCcatd, combmode, mask, t2s, t2sG, stabilize,
             from sklearn.decomposition import PCA
             ppca = PCA(n_components='mle', svd_solver='full')
             ppca.fit(dz)
-            v = ppca.components_
```

Comment:
`u`, `v`, and `s` are pretty standard for an SVD. I'm ok to keep them as is, but if you think this is easier to understand  I'm happy to update !

Reply:
I'd prefer something more readable than single letters, but it doesn't necessarily have to be anything as long as `voxel_comp_weights`.

Reply:
What about `left_singular_vectors`, `right_singular_vectors`, and `singular_values` ? :laughing: 

This is a tough one, really. I usually am strongly against single variable names, but here it's such a strong convention that to break from it is almost confusing.

Maybe we could have @rmarkello or @handwerkerd weigh in.

Reply:
Oof, this is tough. I've struggled with this in my own code that uses SVD, and I eventually settled on abiding by the mathematical conventions (using `u`, `s`, and `v`) with copious comments above the assignment detailing what they represent.

I think @emdupre's [comment below](https://github.com/ME-ICA/tedana/pull/122#discussion_r215481102) about splicing this out into its own function might be the best move!

Reply:
New function (with `u`, `v`, and `s`) created.